modules/vactory_locator/vactory_locator.module

```diff
@@ -427,3 +427,26 @@ function generate_locator_path_alias($locator_storage, $locator_path_alias_orogi
 
   return generate_locator_path_alias($locator_storage, $locator_path_alias_oroginal, $locator_path_alias_oroginal . '-' . $index, $locator_entity, $index + 1);
 }
+
+function vactory_locator_df_jsonapi_output_alter(&$content) {
+  $config = \Drupal::config('vactory_locator.settings');
```

Comment:
@ismailboulaanait 
This hook is invoked for all DFs, so maybe we need to have a check on concerned template before updating the content.
Thanks